index.js

```diff
@@ -2,6 +2,7 @@ var { v4: uuid } = require('uuid');
 var operators = require('./operators');
 var KEY_SELECT = '__key__';
 var KEY_EXISTS = Symbol('KEY_EXISTS');
+var KEY = Symbol('KEY');
```

Comment:
May I suggest calling the actual private instance `KEY_SYMBOL` [as in the real datastore](https://github.com/googleapis/nodejs-datastore/blob/5eccfe608a646eeec0d4b4b96cc7174f931756b6/src/entity.ts#L48)? 

So we would have `Datastore.KEY = KEY_SYMBOL` and `new Datastore().KEY == KEY_SYMBOL`.